content/overview/workflow/edit-overview.md

```diff
@@ -6,37 +6,53 @@ weight: 20
-- [_display-referred workflow_](edit-display-referred.md) is a legacy workflow mode and operates in a similar manner to most other image processing software. In this mode, the tone mapping compression is performed much earlier in the pixelpipe, and many of the modules therefore operate in display-referred space. It uses the [_base curve_](../../module-reference/processing-modules/base-curve.md) module to perform the tone mapping compression.
+4. The [**_white balance_**](../../module-reference/processing-modules/white-balance.md) module does a _primary color grading_, often referred to as _white balancing_. By ensuring the neutral colors (whites and greys) in your image are indeed neutral, it provides a reliable baseline on top of which you can apply creative color gradings. The _white balance_ module normally detects the white balance settings proposed by the camera, however this can be overridden by using sliders or presets that simulate white balance settings in your camera. A more sophisticated workflow is to set the _white balance_ module to "_camera neutral_" and use the [_color calibration_](../../module-reference/processing-modules/color-calibration.md) module to fine-tune the primary grading. This will be done automatically if you set [`preferences > processing > auto-apply chromatic adaptation defaults`](../../preferences-settings/processing.md) to _modern_.
 
-A third option is to set the default workflow to _none_. In this case, the scene-referred workflow module order will be used by default but none of the above modules will be automatically applied. It is up to the user to arrange for appropriate tone mapping and reorder the modules where required. You can change your default workflow in [preferences > processing > auto-apply pixel workflow defaults](../../preferences-settings/processing.md).
+5. The [**_filmic rgb_**](../../module-reference/processing-modules/filmic-rgb.md) module is now used to do a global tone mapping on the image, where we fix up the shadows and highlights in the image. First we set the white relative exposure (the auto picker next to the slider can be helpful if your image contains pure white regions). Adjust the black relative exposure if the blacks are too washed out or too crushed (it is best to do this manually, as the black auto picker can be sensitive to noise and unreliable). The _filmic rgb_ module will set the overall tone mapping for the image; to make finer adjustments to specific areas of the image, we recommend to use the [_tone equalizer_](../../module-reference/processing-modules/tone-equalizer.md) module.
```

Comment:
Perhaps drop "and unreliable"?  The black picker is definitely sensitive to noise.